api/v1alpha1/kafkatopic_types.go

```diff
@@ -46,11 +46,10 @@ type KafkaTopicStatus struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // +k8s:openapi-gen=true
-// +kubebuilder:webhook:failurePolicy="fail",sideEffects="None",name="kafkatopics.kafka.banzaicloud.io",path="/validate",mutating=false,resources={"kafkatopics"},verbs={"create","update"},groups={"kafka.banzaicloud.io"},versions={"v1alpha1"},admissionReviewVersions={"v1"}
-
 // KafkaTopic is the Schema for the kafkatopics API
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+
```

Comment:
Please delete this empty line so the kubebuilder would add the description "KafkaTopic is the Schema for the kafkatopics API" back to the CRD

Reply:
Thank you for pointing on this. I really didn't know why the CRD changed.